src/FileHandler.php

```diff
@@ -109,7 +109,7 @@ public function removeDir($path)
      */
     public function download($zipFile, $source)
     {
-        $client = new Client(['verify' => env('CURL_VERIFY', true)]);
+        $client = new Client(['verify' => config('packager.curl_verify_cert', true)]);
```

Comment:
The true default may be deleted here, since it is defined in the config.

Reply:
Done.

Can you also check this: https://github.com/Jeroen-G/laravel-packager/pull/85?